tools/integration-test/Cargo.toml

```diff
@@ -17,6 +17,7 @@ ibc-relayer-types  = { path = "../../crates/relayer-types" }
 ibc-relayer        = { path = "../../crates/relayer" }
 ibc-test-framework = { path = "../test-framework" }
 
+bincode = "1.0"
```

Comment:
Why is this needed? I cannot find a reference to it anywhere in the code.

Reply:
Oh good catch! This is a leftover from an intermediary solution. I was serializing the `MsgSend` to a `Vec<u8>` inside the test using `bincode::serialize`, I will remove it.